ambuild2/util.py

```diff
@@ -92,11 +92,12 @@ def WaitForProcess(process):
   return process.returncode
 
 def CreateProcess(argv, executable = None):
-  pargs = { 'args': argv }
-  pargs['stdout'] = subprocess.PIPE
-  pargs['stderr'] = subprocess.PIPE
-  if executable != None:
-    pargs['executable'] = executable
+  pargs = {
+    'args': argv,
+    'stdout': subprocess.PIPE,
+    'stderr': subprocess.PIPE,
+    'executable': executable,
```

Comment:
I am aware this isnt the exact thing as it was before, but in this case it doesnt matter, because the default value for `executable` is `None`. So the result will be the same.